src/sql/workbench/services/profiler/browser/profilerService.ts

```diff
@@ -145,12 +152,15 @@ export class ProfilerService implements IProfilerService {
 		return false;
 	}
 
-	public async startSession(id: ProfilerSessionID, sessionName: string): Promise<boolean> {
+	public async startSession(id: ProfilerSessionID, sessionName: string, isSessionTypeLocalFile: boolean = false): Promise<boolean> {
 		if (this._idMap.has(id)) {
 			this.updateMemento(id, { previousSessionName: sessionName });
 			try {
-				await this._runAction(id, provider => provider.startSession(this._idMap.get(id)!, sessionName));
-				this._sessionMap.get(this._idMap.reverseGet(id)!)!.onSessionStateChanged({ isRunning: true, isStopped: false, isPaused: false });
+				await this._runAction(id, provider => provider.startSession(this._idMap.get(id)!, sessionName, isSessionTypeLocalFile));
+				if (!isSessionTypeLocalFile) {		// Do nothing for XEL file read
```

Comment:
Same here, we should still do this (and then our MSSQL implementation can just have it be a no-op if that makes the most sense)